packages/create-app/src/generate/generateNpmRc.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export function generateNpmRc({
+  osdkPackage,
+  osdkRegistryUrl,
+}: {
+  osdkPackage: string;
+  osdkRegistryUrl: string;
+}): string {
+  const withoutProtocol = osdkRegistryUrl.replace(/^https:\/\//, "");
```

Comment:
I guess this is safe given `promptOsdkRegistryUrl` already tests it against a regex. If we plan to use this somewhere that isn't part of this code path, should we consider passing the protocol but have it with a default value?

Reply:
I think the only other protocol that would be relevant would be `http` but should be safe to only allow `https` for this url